tools/staticbuild/README.md

```diff
@@ -45,6 +44,7 @@ CMAKE_STATICBUILD=1 tools/staticbuild/build.sh cpu
 
 For the CMake build, you need to install `patchelf` first, for example via `apt
 install patchelf` on Ubuntu systems.
+>>>>>>> ae145cdb1de95e8275dd087585f4aab69bc94005
```

Comment:
seems to be a leftover from resolving merge conflict

Reply:
Thanks. It's fixed.